### PR TITLE
Feature/analyses 3-digit evs_ver

### PR DESCRIPTION
**Pull request by PerryShafran-NOAA**
## Pull Request Testing ##

- [ ] Describe testing already performed for this Pull Request:</br>

Put in evs_ver=v1.0.0, the 3-digit version number required by EE2.  To ensure a 2-digit version number for COMIN and COMOUT, we used cut to shorten and created a new variable for the driver only, evs_ver_2d

- [ ] Recommend testing for the reviewer(s) to perform, including the location of input datasets, and any additional instructions:</br>

You can test one or more of the stats drivers.  Please be sure you include -v vhr=00 in your qsub command, or any cycle you choose.

For plots, set COMIN to emc.vpppg.  

- [ ] Has the code been checked to ensure that no errors occur during the execution? **[Yes or No]**

- [ ] Do these updates/additions include sufficient testing updates? **[Yes or No]**

- [ ] Please complete this pull request review by **[Fill in date]**.</br>

ASAP

## Pull Request Checklist ##

- [ ] Review the source issue metadata (required labels, projects, and milestone).
- [ ] Complete the PR description above.
- [ ] Ensure the PR title matches the feature branch name.
- [ ] Check the following:
- [ ]  Instructions provided on how to run
- [ ]  Developer's name is replaced by ${user} where necessary throughout the code
- [ ]  Check that the ecf file has all the proper definitions of variables
- [ ]  Check that the jobs file has all the proper settings of COMIN and COMOUT and other input variables
- [ ]  Check to see that the output directory structure is followed
- [ ]  Be sure that you are not using MET utilities outside the METplus wrapper structure

- [ ] After submitting the PR, select **Development** issue with the original issue number.
- [ ] After the PR is approved, merge your changes. If permissions do not allow this, request that the reviewer do the merge.
- [ ] Close the linked issue.

### CODE REVIEW

**Comment by ShelleyMelchior-NOAA** [comment]
I have cloned your fork and checked out branch `feature/analyses_run_ver`.

File: dev/drivers/scripts/stats/analyses/jevs_analyses_urma_grid2obs_stats.sh [hunk 1]
```diff
@@ -9,19 +9,23 @@
...
+set -x
+
 export model=evs
 
 export HOMEevs=/lfs/h2/emc/vpppg/noscrub/$USER/EVS
 
 source $HOMEevs/versions/run.ver
 
+export evs_ver=v1.0.0
+evs_ver_2d=$(echo $evs_ver | cut -d'.' -f1-2)f1
```
Comment (ShelleyMelchior-NOAA) [inline dev/drivers/scripts/stats/analyses/jevs_analyses_urma_grid2obs_stats.sh]: Please remove the typo at the end of line 21.

Comment (AliciaBentley-NOAA) [reply dev/drivers/scripts/stats/analyses/jevs_analyses_urma_grid2obs_stats.sh]: @PerryShafran-NOAA If you can't remember off the top of your head (I know I can't), this line should be:
`evs_ver_2d=$(echo $evs_ver | cut -d'.' -f1-2)`

Comment (PerryShafran-NOAA) [reply dev/drivers/scripts/stats/analyses/jevs_analyses_urma_grid2obs_stats.sh]: Typo removed, and confirmed that it didn't make it to the other stats script as I did a cut 'n' paste.

Comment (PerryShafran-NOAA) [reply dev/drivers/scripts/stats/analyses/jevs_analyses_urma_grid2obs_stats.sh]: @AliciaBentley-NOAA thanks!  I knew the line; that was a legit typo.  ;)

**Comment by ShelleyMelchior-NOAA** [comment]
I have run the stats processing.
COMOUT=`/lfs/h2/emc/vpppg/noscrub/shelley.melchior/evs/v1.0/stats/analyses`
logfiles: `/lfs/h2/emc/ptmp/shelley.melchior/EVS_out/`
`jevs_analyses_rtma_ru_grid2obs_stats.o106893352`
`jevs_analyses_urma_grid2obs_stats.o106893799`
`jevs_analyses_rtma_grid2obs_stats.o106893357`

**Comment by PerryShafran-NOAA** [comment]
I see that the code has made evs_ver the three digit version, then became the 2 digit version, and thus COMIN and COMOUT were properly set.  Everything else should act exactly the same as before.  So that stats jobs are good.

**Comment by ShelleyMelchior-NOAA** [comment]
Great!
Plots processing has concluded.
COMOUT=`/lfs/h2/emc/vpppg/noscrub/shelley.melchior/evs/v1.0/plots/analyses/atmos.20231019`
logfile: `/lfs/h2/emc/ptmp/shelley.melchior/EVS_out/jevs_analyses_grid2obs_plots.o106894629`
The tar file is not empty and the evs_ver, evs_ver_2d and COMIN and COMOUT vars are properly resolved.
```
$ grep evs_ver jevs_analyses_grid2obs_plots.o106894629
++ export evs_ver=v1.0
++ evs_ver=v1.0
++ export ve_evs_ver=1.0
++ ve_evs_ver=1.0
+ export evs_ver=v1.0.0
+ evs_ver=v1.0.0
+ evs_ver_2d=v1.0
export evs_ver=v1.0.0
evs_ver_2d=$(echo $evs_ver | cut -d'.' -f1-2)
```
```
$ grep COMOUT jevs_analyses_grid2obs_plots.o106894629
+ export COMOUT=/lfs/h2/emc/vpppg/noscrub/shelley.melchior/evs/v1.0
+ COMOUT=/lfs/h2/emc/vpppg/noscrub/shelley.melchior/evs/v1.0
```
```
$ grep COMIN jevs_analyses_grid2obs_plots.o106894629                                                                         
+ export COMIN=/lfs/h2/emc/vpppg/noscrub/emc.vpppg/evs/v1.0
+ COMIN=/lfs/h2/emc/vpppg/noscrub/emc.vpppg/evs/v1.0
```

**Comment by PerryShafran-NOAA** [comment]
The plot tarball is identical in size to the one in the emc.vpppg parallel.

I see that the version number is properly resolved.

I think we are good to merge this one in.

**Comment by ShelleyMelchior-NOAA** [comment]
I concur. I will do the formal code review and approve the PR for merge.

**Review by ShelleyMelchior-NOAA — APPROVED** [review]
All tests yielded expected results. This PR is approved for merge.

File: dev/drivers/scripts/plots/analyses/jevs_analyses_grid2obs_plots.sh [hunk 2]
```diff
@@ -8,16 +8,20 @@
...
+set -x
+
 export model=evs
 
 export HOMEevs=/lfs/h2/emc/vpppg/noscrub/$USER/EVS
 
 source $HOMEevs/versions/run.ver
 
+export evs_ver=v1.0.0
+evs_ver_2d=$(echo $evs_ver | cut -d'.' -f1-2)
```
Comment (YaliMao-NOAA) [inline dev/drivers/scripts/plots/analyses/jevs_analyses_grid2obs_plots.sh:20]: To my understanding, we don't need a 2d version of evs_ver. compath.py will return a 2d version.

**Comment by AliciaBentley-NOAA** [comment]
@YaliMao-NOAA compath.py (without the -o) only works on models that are currently in operations. Since EVS is not currently in operations, compath.py does not work on it. We sometimes need to define COMIN and COMOUT in our dev drivers with a 2-digit version of $evs_ver. Does that make sense?

**Comment by YaliMao-NOAA** [comment]
@AliciaBentley-NOAA 
I did a test:
export COMPATH=/lfs/h2/emc/vpppg/noscrub/anyone/anyfolder/com/evs
compath.py com/evs/v1.0.0
It returns /lfs/h2/emc/vpppg/noscrub/anyone/anyfolder/com/evs/v1.0, though the folder doesn't exist.

**Comment by AliciaBentley-NOAA** [comment]
@YaliMao-NOAA You are correct. The issue is that we do not have a /com directory in the emc.vpppg parallel directory structure. I believe that this is what prevents us from using compath.py that way. 

EVSv1 parallel prep output: /lfs/h2/emc/vpppg/noscrub/emc.vpppg/evs/v1.0/prep
EVSv1 parallel stats output: /lfs/h2/emc/vpppg/noscrub/emc.vpppg/evs/v1.0/stats
EVSv1 parallel plots output: /lfs/h2/emc/ptmp/emc.vpppg/evs/v1.0/plots

**Comment by YaliMao-NOAA** [comment]
That's true.
> The issue is that we do not have a /com directory

**Pull request merged**